pkg/metrics/metrics.go

```diff
@@ -228,13 +228,13 @@ The label 'underlying_cause' can have the following values:
 		}, []string{"cluster_queue", "priority_class"},
 	)
 
-	localQueueAdmissionWaitTime = prometheus.NewHistogramVec(
+	LocalQueueAdmissionWaitTime = prometheus.NewHistogramVec(
 		prometheus.HistogramOpts{
 			Subsystem: constants.KueueName,
 			Name:      "local_queue_admission_wait_time_seconds",
 			Help:      "The time between a workload was created or requeued until admission, per 'local_queue'",
 			Buckets:   generateExponentialBuckets(14),
-		}, []string{"name", "namespace"},
+		}, []string{"name", "namespace", "priority_class"},
```

Comment:
nit: @vladikkuzn within the context of this PR and the other related priority_class metrics ones. Could you name it to `workload_priority_class`

I know that this metric itself is only contextualized with workload. But when we mix it in production dashboards, there's various kinds of priorities that would be difficult for engineers to differentiate.

Reply:
I see, thanks for the link! Is there a follow up issue to add the source?